test/plugins/sensor_demo/custom_sensor/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+cmake_minimum_required(VERSION 3.10.2 FATAL_ERROR)
+
+project(custom_rendering_sensor)
+
+find_package(ignition-cmake2 REQUIRED)
+find_package(ignition-sensors6 REQUIRED)
+find_package(ignition-rendering6 REQUIRED)
+find_package(ignition-msgs8 REQUIRED)
+find_package(ignition-transport11 REQUIRED)
+find_package(ignition-common4 REQUIRED)
+find_package(ignition-math6 REQUIRED)
+find_package(ignition-sensors6-rendering REQUIRED)
```

Comment:
Since this `CMakeLists.txt` is being included by the top level `CMakeLists.txt`, I don't think it's necessary to do `find_package` here. All the packages should already be found at this point. Using explicit versions also makes this harder to maintain as we move to newer versions. Instead, I'd suggest removing this and in `target_link_libraries` use the packages already found. eg. Instead of `ignition-sensors6::ignition-sensors6`, do `ignition-sensors${IGN_SENSORS_VER}::ignition-sensors${IGN_SENSORS_VER}`

Reply:
Done !